src/iam/certmodules/pkcs11/pkcs11.cpp

```diff
@@ -18,6 +18,31 @@ namespace aos {
 namespace iam {
 namespace certhandler {
 
+/***********************************************************************************************************************
+ * Static
+ **********************************************************************************************************************/
+
+static Error IDToStr(const Array<uint8_t>& id, String& idStr)
```

Comment:
I would suggest renaming to EncodePKCS11ID

Reply:
done